python-backend/tests/status/resources/test_mine_status_resource.py

```diff
@@ -1,19 +1,17 @@
 import json, uuid
 
-from app.api.constants import MINE_STATUS_OPTIONS
-
 
 # GET
 def test_get_mine_status_option(test_client, db_session, auth_headers):
     get_resp = test_client.get('/mines/status', headers=auth_headers['full_auth_header'])
     get_data = json.loads(get_resp.data.decode())
-    assert get_data == {'options': MINE_STATUS_OPTIONS}
+    assert len(get_data.get('records')) == 14
```

Comment:
Not that it should really matter, since the test will fail either way, but I think directly accessing the key is acceptable in the tests (ie. `get_data['records']`). This helps to also verify that the `records` attribute is returned in the response object

Reply:
Changed